docs/source/_templates/partials/submitting_to_validator.rst

```diff
@@ -43,6 +43,18 @@ prepared the BatchList:
         bytes.NewBuffer(batchListBytes)
     )
 
+{% elif language == 'Rust' %}
+
+.. code-block:: rust
+
+    extern crate reqwest;
+
+    let client = reqwest::Client::new();
+    let res = client.post("http://localhost:8008/batches")
```

Comment:
[nitpick] This PR about documenting, 2 things that would make it look more nice.
1. It would be good if all language tutorial have same examples, same variable names, ip address, strings etc. It'll help beginners who know one language and want to switch to another language. There will be less ambiguity and tutorial can be easily mapped.
2. Writing a sample code, then copying snippets from editor would make it look more good in documentation. Example a space after ',' in line 54.

Reply:
1. I did try to do that, but I'll look for places where I miss it
2. I did write a sample code, but in vim, without any "pretiffy" kind of plugin, but I can run I'll look into this